cypress/support/commands.ts

```diff
@@ -0,0 +1,26 @@
+// https://on.cypress.io/custom-commands
+import 'cypress-wait-until';
+import 'cypress-file-upload';
+
+import { expect } from 'chai';
```

Comment:
se puede usar el `expect` de [jest](https://jestjs.io/docs/expect)? así no necesitamos _también_ traernos todo chai y mocha como dependencias.

Reply:
https://github.com/cypress-io/cypress/issues/281#issuecomment-387623959 
Parece que no esta totalmente soportado. Ellos recomiendan usar chai y mocha
https://docs.cypress.io/guides/references/bundled-tools#Mocha

Reply:
pero sí se puede usar el `expect` de jest! https://jestjs.io/docs/api

```suggestion
import { expect } from '@jest/globals';
```

Reply:
![image](https://user-images.githubusercontent.com/74751751/147368808-f35a659c-77a8-4799-89f4-8f5159bf1ec2.png)

 Y cuando lo hago sin el `import` 
![image](https://user-images.githubusercontent.com/74751751/147368824-6578f780-dd17-4ede-89af-0c919db31d15.png)

Creo que jest y chai ya vienen dentro del bundle de cypress. Investigando un poco, solo tenia que haber instalado los types, no todo el paquete. Deja investigo y hago pruebas

Reply:
Si fue asi. Mocha y chai ya vienen dentro de cypress por defecto, solo era necesario instalar los tipos. Para evitar overlap entre los tipos de jest y de chai, mejor cree un `tsconfig` exclusivo para cypress dentro de `./cypress` y exclui el directorio del `tsconfig` root.